src/app/search/state/search.query.ts

```diff
@@ -32,6 +32,7 @@ export class SearchQuery extends Query<SearchState> {
   public autoCompleteTerms$: Observable<Array<string>> = this.select(state => state.autocompleteTerms);
   public hasAutoCompleteTerms$: Observable<boolean> = this.autoCompleteTerms$.pipe(map(terms => terms.length > 0));
   public suggestTerm$: Observable<string> = this.select(state => state.suggestTerm);
+  public pageSize: Observable<number>=this.select(state => state.pageSize);
```

Comment:
This should be `public pageSize$...` to follow the convention of variables of type Observable ending with a dollar sign.

Also the formatting around the equals sign is not consistent.